.circleci/Dockerfile

```diff
@@ -38,10 +38,9 @@ RUN     cd /tmp && \
         sudo mv etcd-${ETCD_VER}-linux-amd64/etcd /usr/local/bin/etcd && \
         rm -rf etcd*
 
-RUN     mkdir -p /tmp/apiserver && \
-        cd /tmp/apiserver && \
-        curl -L -o kube-apiser https://storage.googleapis.com/kubernetes-release/release/${K8S_VER}/bin/linux/amd64/kube-apiserver && \
-        chmod +x /tmp/apiserver/kube-apiserver
+RUN     curl -L -o /tmp/kube-apiser https://storage.googleapis.com/kubernetes-release/release/${K8S_VER}/bin/linux/amd64/kube-apiserver && \
+        chmod +x /tmp/kube-apiserver && \
```

Comment:
this logic doesn't make any sense to me.  /tmp/kube-apiserver is presented in the dockerfile where?  Is the intended location /usr/local/bin?  Why curl twice (this is a second curl in - first commit = first curl) and not just use cp -a?